### PR TITLE
Ros2 basic day2

### PR DESCRIPTION
**Pull request by DavidMerzJr**
Fix Exercise 3 Moveit syntax to ROS2, among other improvements.

### CODE REVIEW

**Comment by jdlangs** [comment]
@DavidMerzJr this is now fully updated and tested

@JeremyZoss this took quite a while to finalize but this is now the fully working MoveIt2 exercise in Foxy. We'll have to merge tonight for tomorrow's training but feel free to take a look and drop any comments if you have some availability.

File: .travis.yml [hunk 1]
```diff
@@ -21,12 +21,12 @@ matrix:
...
     - CONTAINER_IMAGE=ros:melodic \
       CONTAINER_NAME=industrial_training_ci \
-      SRC_DIRS="exercises/3.0/src exercises/3.1/src exercises/3.2/src exercises/3.3/src"
+      SRC_DIRS="exercises/3.0/ros1/src exercises/3.1/ros1/src exercises/3.2/ros1/src exercises/3.3/ros1/src"
   - name: "Exercise 4"
     env:
     - CONTAINER_IMAGE=ros:melodic \
       CONTAINER_NAME=industrial_training_ci \
-      SRC_DIRS="exercises/4.0/src exercises/4.1/src"
+      SRC_DIRS="exercises/4.0/ros1/src exercises/4.1/ros1/src"
```
Comment (DavidMerzJr) [inline .travis.yml:29]: I think that maintaining both ROS1 and ROS2 in the same branch is getting out of hand - updating these two exercises (mostly) to ROS2 generated 281 file changes.  I'm not sure how many people have strong opinions about this, though.

Comment (jdlangs) [reply .travis.yml:29]: I'm not following why you're suggesting there's a problem. I'm not aware of another option that would be less disruptive.

**Comment by jdlangs** [comment]
Hitting merge to get readthedocs building...

**Pull request merged**